Tutorials/03-Wifi_Getting_Started_Guide.md

```diff
@@ -1,161 +1,153 @@
+ * Learn about the physical hardware on the WiFi Shield checkout the [OpenBCI Wifi Hardware docs](http://docs.openbci.com/Hardware/11-Wifi)
+ * Lookup the commands for the HTTP rest server on the WiFi Shield at the [swaggerhub.io](https://app.swaggerhub.com/apis/pushtheworld/openbci-wifi-server/1.3.0)
+ * Checkout the firmware that powers the WiFi Shield at [on Github](http://github.com/OpenBCI/OpenBCI_WIFI)
 
-![Battery Connection](../assets/images/wifi_ganglion_power.jpg)
+Always make sure you have the [latest firmware](https://github.com/OpenBCI/OpenBCI_WIFI/releases/latest) loaded on your WiFi Shield. Additionally, you will need to update your [Cyton (at least version v3.0.0-rc4 or newer)](https://github.com/OpenBCI/OpenBCI_32bit_Library/releases/tag/v3.0.0-rc4) to the latest firmware and/or [Ganglion (at least v2.0.0-rc1 or newer)](https://github.com/OpenBCI/OpenBCI_Ganglion_Library/releases/tag/v2.0.0-rc1).
 
-Install 4 AA batteries in your battery pack or charge up that LiPo battery. The Wifi shield has a 4-5x larger power draw then the Bluetooth communication system so we recommend using LiPo to save! As of today, the Wifi shield cannot pass through power to the Ganglion. **Keep external `EXT PWR` to `OFF` when using Ganglion**
+This guide will walk you through setting up your WiFi Shield, connecting it to your computer, and then connecting it to yourself. The first tutorial is for the Ganglion and the second is for the Cyton.
 
-### Seating the Wifi Shield
+## Powering Your WiFi Shield
 
-Remove power to your Ganglion and Wifi shield.
+A 3.7V LiPo rechargeable battery is highly recommended for powering the WiFi Shield.
 
-First line up the big header.
+### Cyton
 
-![Line up the big header](../assets/images/wifi_ganglion_seating_1.jpg)
+You only need one battery when using the Cyton with the WiFi Shield. The Cyton takes power from the WiFi Shield. When the Cyton is powered by the WiFi Shield, you'll find the power switch on the Cyton becomes useless. Don't worry, that's intended. More about powering the Cyton and WiFi Shield can be found in the [these detailed docs](http://docs.openbci.com/Hardware/11-Wifi).
 
-Then line up the two four pin headers.
-
-![Line up the other pins](../assets/images/wifi_ganglion_seating_2.jpg)
-
-Finally once all the pins are lined up, you can press straight down to fully seat the wifi shield onto it's new Ganglion home :)
-
-![Press down to seat](../assets/images/wifi_ganglion_seating_3.jpg)
-
-### Removing the Wifi Shield
-
-Remove power to your Ganglion and/or Wifi shield.
-
-![Wifi remove even fingers](../assets/images/wifi_ganglion_removing_1.jpg)
+![OpenBCI Wifi Contents](../assets/images/wifi_what_you_need.jpg)
 
-Wiggle the board slowly off
+#### What you need
 
-![Wifi wiggle](../assets/images/wifi_ganglion_removing_2.jpg)
+1. OpenBCI WiFi Shield
```

Comment:
Somewhere here you should have the steps laid out in bullet form

- Turn off power on your board and the shield
- Plug the shield on the board
- Turn on the shield
- Turn on the board

or whatever order of operations is correct